survey_sign_oca/views/res_config_settings_view.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!-- Copyright 2025 Kencove - Mohamed Alkobrosli
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
+<odoo>
+    <record id="res_config_settings_view_form" model="ir.ui.view">
+        <field name="name">res.config.settings.view.form.inherit</field>
+        <field name="model">res.config.settings</field>
+        <field name="priority" eval="93" />
+        <field name="inherit_id" ref="base.res_config_settings_view_form" />
```

Comment:
The proper thing to do would be to use the Survey > Settings option which does not exist, using a new block I think is not the most appropriate, IMO there are 2 possible options:

A- Add those fields to Sign > General settings, inheriting from `sign_oca.res_config_settings_view_form`.
B- Create a `base_survey_config` module similar to https://github.com/OCA/maintenance/tree/16.0/base_maintenance_config to add a Survey > Settings menu and have this module depend on the other one.

I think the easiest option in this case is option A.